platforms/minestom/src/main/java/com/dfsek/terra/minestom/entity/DeferredMinestomEntity.java

```diff
@@ -0,0 +1,68 @@
+package com.dfsek.terra.minestom.entity;
+
+import com.dfsek.terra.api.entity.Entity;
+import com.dfsek.terra.api.entity.EntityType;
+import com.dfsek.terra.api.util.vector.Vector3;
+import com.dfsek.terra.api.world.ServerWorld;
+import com.dfsek.terra.minestom.world.TerraMinestomWorld;
+
+import net.minestom.server.coordinate.Pos;
+
+
+public class DeferredMinestomEntity implements Entity {
```

Comment:
Why is this needed?

Reply:
During World generation, no entities can be placed in minestom. This uses the async chunk loading api to create the entities after the world is loaded